homework/App.js

```diff
@@ -12,18 +12,35 @@ class App {
    * @param {string} url The GitHub URL for obtaining the organization's repositories.
    */
   async initialize(url) {
-    // Add code here to initialize your app
-    // 1. Create the fixed HTML elements of your page
-    // 2. Make an initial XMLHttpRequest using Util.fetchJSON() to populate your <select> element
-
     const root = document.getElementById('root');
-
-    Util.createAndAppend('h1', root, { text: 'It works!' }); // TODO: replace with your own code
+    const header = Util.createAndAppend('header', root, { class: 'header' });
+    Util.createAndAppend('h3', header, { text: 'HYF Repositories' });
+    const selectMenu = Util.createAndAppend('select', header, {
+      class: 'repo-selector',
+      tabindex: 1,
```

Comment:
If you want to set the focus on the `<select>` element when the page loads,  setting the `tabindex` doesn't work. Use `autofocus` instead:

```js
 class: 'repo-selector',
 autofocus: true,
```

https://developer.mozilla.org/en-US/docs/Web/HTML/Element/select#Attributes

Reply:
I removed lowercase thing. it works